Cargo.toml

```diff
@@ -7,24 +7,45 @@ autobins = false
 [dependencies]
 chrono = "0.4"
 url = "1.4"
-futures-preview = { version = "0.2", optional = true }
-atk = "^0"
-glib-sys = "^0"
-gobject-sys = "^0"
-glib = "^0"
-gio = "^0"
-gdk = "^0"
-gdk-pixbuf = "^0"
-gtk = "^0"
-pango = "^0"
-cairo-rs = { version = "^0", features = ["png"] }
+futures-preview = { version = "0.3.0-alpha.15", optional = true}
```

Comment:
This is wrong change

Reply:
I meant: master need use packages, not gits

Reply:
A surprising one for sure.

Reply:
Don't worry, I got it. ;)